code/_onclick/hud/plane_master.dm

```diff
@@ -39,6 +39,18 @@
 /atom/movable/screen/plane_master/proc/clear_filters()
 	filters = list()
 
+///Blur rendered on z-levels below.
+/atom/movable/screen/plane_master/openspace_blur
+    name = "open space blur plane master"
```

Comment:
Blur is the effect you are applying. The plane master's name should be more generic. `openspace` would suffice to describe it.

Reply:
That currently exists and is occupied by the code that does the drop shading, though it uses its own backdrop plane, not the actual openspace plane.
I had trouble trying to implement the blur there.

Reply:
Rename the other one then. This is a bad and actively misleading name.

Reply:
OK, done.